compat/src/index.js

```diff
@@ -194,51 +177,6 @@ let Children = {
 	toArray: toChildArray
 };
 
-/**
- * Wrap `createElement` to apply various vnode normalizations.
- * @param {import('./internal').VNode["type"]} type The node name or Component constructor
- * @param {object | null | undefined} [props] The vnode's properties
- * @param {Array<import('./internal').ComponentChildren>} [children] The vnode's children
- * @returns {import('./internal').VNode}
- */
-function createElement(...args) {
-	let vnode = h(...args);
-
-	let type = vnode.type,
-		props = vnode.props;
-	if (typeof type != 'function') {
-		if (props.defaultValue) {
-			if (!props.value && props.value !== 0) {
-				props.value = props.defaultValue;
-			}
-			delete props.defaultValue;
-		}
-
-		if (Array.isArray(props.value) && props.multiple && type === 'select') {
-			toChildArray(props.children).forEach(child => {
-				if (props.value.indexOf(child.props.value) != -1) {
-					child.props.selected = true;
-				}
-			});
-			delete props.value;
-		}
-		handleElementVNode(vnode, props);
-	}
-
-	vnode.preactCompatNormalized = false;
-	return normalizeVNode(vnode);
-}
-
-/**
- * Normalize a vnode
- * @param {import('./internal').VNode} vnode
- */
-function normalizeVNode(vnode) {
-	vnode.preactCompatNormalized = true;
```

Comment:
Just calling that I removed this property cuz I don't think it's being used anymore. Holler if you think this is worth the 16 bytes.

Reply:
Iirc this is a leftover from earlier iterations. Previous versions would check for this property and only normalize a `vnode` if it was missing.